pom.xml

```diff
@@ -1144,9 +1144,9 @@
         <scope>test</scope>
       </dependency>
       <dependency>
-        <groupId>com.novocode</groupId>
+        <groupId>com.github.sbt</groupId>
         <artifactId>junit-interface</artifactId>
-        <version>0.11</version>
+        <version>0.13.3</version>
```

Comment:
It seems that the setup guide is outdated because it still points to `0.13.2`.

Reply:
ok ~ I'll try to promote the update of setup